### ISSUE TITLE
misprint

### ISSUE THREAD

**Issue by perixx1234** [ngrx/platform#2139]
/guide/entity/adapter => user.reducer.ts => on(UserActions.upsertUsers, (state, { users }) => {
    return adapter.upsertUsers(users, state);
  }),

"adapter.upsertUsers(users, state)" should be "adapter.upsertMany(users, state)" ???

**Comment by timdeschryver** [comment]
Yes, you're right - thanks!
Do you want to create a Pull Request for this?

**Comment by perixx1234** [comment]
No, thanks)

**Comment by tashoecraft** [comment]
line 157 for projects/ngrx.io/content/guide/entity/adapter.md on master reads
```
  on(UserActions.upsertUsers, (state, { users }) => {
    return adapter.upsertMany(users, state);
  }),
```
Am I missing something? Or was this resolved in the past 5 hours without reference?

**Comment by timdeschryver** [comment]
My bad, this was already resolved in #2105, but the website has not been released.
